remotespark/__init__.py

```diff
@@ -1,2 +1 @@
-from .RemoteSparkMagics import *
```

Comment:
This was just to make sure that everything gets imported and is recognized by IPython. I'm not sure if it's still necessary for newer versions of IPython.